chickenRice.html

```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>The Best Chicken Rice Recipes!</title>
+  </head>
+  <body>
+    <h1>Chicken Rice</h1>
+
+    <h3> Ingredients </h3>
+        <ul>
+            <li>Chicken</li>
+            <li>Soup</li>
+            <li>Rice</li>
+            <li>Garlic Ginger Sauce</li>
+            <li>Red Chili Sauce</li>
+        </ul>
```

Comment:
Indentation looks a bit off here. Do you have auto-format on save for HTML? If not, might be worth looking into it!